src/main/java/com/esri/samples/raster/raster_rendering_rule/README.md

```diff
@@ -0,0 +1,39 @@
+<h1>Raster Rendering Rule</h1>
+
+<p>Display a raster on a map and apply different rendering rules to that raster.</p>
+
+<p><img src="RasterRenderingRule.png"/></p>
+
+<h2>Use case</h1>
+
+<p>Image Service Rasters may provide several rendering rules options. When it is necessary to switch between rendering rules in order to display the desired data, the interface for the selection and application of the required rule can be used.</p>
+
+<h2>How to use the sample</h2>
+
+<p>Run the sample and use the drop-down menu at the top to select a rendering rule.</p>
+
+<h2>How it works</h2>
+
+<ol>
+  <li> Create an <code>ImageServiceRaster</code> and add it to a <code>RasterLayer</code> and add the <code>RasterLayer</code> to the map as an operational layer. </li>
+  <li> Connect to the <code>loadStatusChanged</code> signal for the image service raster.</li>
```

Comment:
A `signal` is Qt-specific. In the Java API, this would be a callback.

This section is supposed to be higher-level than that though. You can simply write "load the image service raster."